### PR TITLE
added 2nd method

### PR DESCRIPTION
**Pull request by 4teenth**

### CODE REVIEW

File: contacts.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  getContactById = async id => {
+    const contactsData = await this.getListContacts();
+    return contactsData.find(contact => contact.id === id);
+  };
+
+  removeContactById = async id => {
+    const contactsData = await this.getListContacts();
+    const result = contactsData.filter(contact => contact.id !== id);
+    fs.writeFile(this.contactsPath, JSON.stringify(result));
```
Comment (odykyi) [inline contacts.js]: await fs.writeFile(this.contactsPath, JSON.stringify(result));

File: contacts.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    const id = contactsData.length ? [...contactsData].pop().id + 1 : 1;
+    const newContact = {
+      id,
+      name,
+      email,
+      phone,
+    };
+    contactsData.push(newContact);
+    const contactsDataAsJSON = JSON.stringify(contactsData);
+    fs.writeFile(this.contactsPath, contactsDataAsJSON);
```
Comment (odykyi) [inline contacts.js]: await

File: contacts.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    return newContact;
+  };
+}
+
+module.exports = new Contacts();
+
+// Method#2++++++++++++++++++++++++++++++++++++++
```
Comment (odykyi) [inline contacts.js]: remove commented code

File: index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      break;
+
+    default:
+      console.warn('\x1B[31m Unknown action type!');
+  }
+}
+
+invokeAction(argv);
+
+// Method#2++++++++++++++++++++++++++++++++++
```
Comment (odykyi) [inline index.js]: remove commented

**Review by odykyi — CHANGES_REQUESTED** [review]
requested changes

**Comment by 4teenth** [comment]
Got it! Fixed

**Review by odykyi — APPROVED** [review]

**Pull request merged**